alert-common/src/main/java/com/synopsys/integration/alert/common/persistence/accessor/JobAccessor.java

```diff
@@ -18,6 +18,9 @@
 import com.synopsys.integration.alert.common.rest.model.AlertPagedModel;
 
 public interface JobAccessor {
+    
+    int countJobsByFrequency(String frequency);
```

Comment:
Do we want a count, or just a boolean? EX: hasJobsWithFrequency

Reply:
changed